src/Whoops/Resources/css/whoops.base.css

```diff
@@ -461,6 +461,36 @@ kbd {
 }
 
 pre.sf-dump {
-    padding: 0px !important;
-    margin: 0px !important;
-}
\ No newline at end of file
+  padding: 0px !important;
+  margin: 0px !important;
+}
+
+.icon-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 10px 0;
+}
+  .icon-list li {
+    float: left;
```

Comment:
I wouldn't make you make this bigger change usually, but in this case it will be also beneficial for you.
Forget floats, use inline-blocks. [Read more](http://stackoverflow.com/questions/15172520/advantages-of-using-displayinline-block-vs-floatleft-in-css).